src/test/java/org/folio/rest/RestTestSuite.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.rest;
+
+import org.folio.rest.impl.StorageHelperTest;
+import org.folio.rest.impl.TenantRefAPITest;
+import org.folio.rest.support.CollectionUtilTest;
+import org.folio.rest.support.CompletableFutureUtilTest;
+import org.folio.rest.support.CqlQueryTest;
+import org.folio.rest.unit.ItemDamagedStatusAPIUnitTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+  CollectionUtilTest.class,
+  CompletableFutureUtilTest.class,
+  CqlQueryTest.class,
+  ItemDamagedStatusAPIUnitTest.class,
+  StorageHelperTest.class,
+  TenantRefAPITest.class,
+})
```

Comment:
I was being a bit excessively paranoid about controlling the order to help avoid the random failures rather early on in the process of writing the changes reflected in this PR.

Given that a new strategy is now proposed to avoid excessive Suite usage, I partially reverted the suite changes to avoid adding code that we will immediately remove in a subsequent PR.